api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirTaskServiceImpl.java

```diff
@@ -85,7 +86,7 @@ public Task updateTask(String uuid, Task task) {
 		FhirTask openmrsTask = null;
 		
 		if (uuid != null) {
-			openmrsTask = dao.getTaskByUuid(task.getId());
+			openmrsTask = dao.get(task.getId());
```

Comment:
You should actually be able to remove the `updateTask` function altogether and switch to using the default `update` function.

Reply:
I tried removing the whole updateTask function like how you suggested  class but same error appeared  cc @ibacher